test-integ/peering_commontopo/README.md

```diff
@@ -1,12 +1,66 @@
-# peering_commontopo
+# CONSUL PEERING COMMON TOPOLOGY TESTS
 
 These peering tests all use a `commonTopo` (read: "common topology") to enable sharing a deployment of a Consul. Sharing a deployment of Consul cuts down on setup time.
 
-This is only possible if two constraints are followed:
+To run these tests, you will need to have docker installed. Next, make sure that you have all the required consul containers built:
 
+```
+make test-compat-integ-setup
+```
+
+## Non-Shared CommonTopo Tests
+
+The tests in question are designed in a manner that modifies the topology. As a result, it is noy possible to share the testing environment across these tests.
+
+## Shared CommonTopo Tests
+
+The tests in question are designed in a manner that does not modify the topology. As a result, it is possible to share the testing environment across these tests.
```

Comment:
```suggestion
The tests in question are designed in a manner that does not modify the topology in any way that would interfere with other tests. As a result, it is possible to share the testing environment across these tests.
```

Not exactly sure how to word this, but they do modify the topology in their `setup`. Maybe "during their `test` phase?